doc/nrf/dev_model_and_contributions/code_base.rst

```diff
@@ -151,28 +151,32 @@ The |NCS| version numbers are assigned based on very specific criteria.
-|                            | Indicates a snapshot of the development state that is created to highlight | The ``-devN`` version might not be subjected to the same amount of testing as a regular |NCS| release.                               |
-|                            | the introduction of a new piece of functionality.                          |                                                                                                                                      |
-+----------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
+|        Version                 |                              Description                                                                                             |
++================================+======================================================================================================================================+
+| ``MAJOR``                      | Introduces a large number of substantial changes across the board.                                                                   |
+|                                | Major version number is increased rarely.                                                                                            |
+|                                | Major releases are always accompanied by a :ref:`migration guide <migration_guides>`.                                                |
++--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``MINOR``                      | Introduces one or more new functionalities.                                                                                          |
+|                                | Version number is increased every time a minor release is cut.                                                                       |
+|                                | Minor releases are the default types of an |NCS| release and usually are accompanied by a :ref:`migration guide <migration_guides>`. |
+|                                | The new functionalities may break existing APIs.                                                                                     |
++--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| ``PATCH``                      | Introduces bugfixes or minor changes to the existing functionalities.                                                                |
```

Comment:
```suggestion
| ``PATCH``                      | Introduces bug fixes or minor changes to the existing functionalities.                                                                |
```
As per [Terms](https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120298016/Terms+abbreviations+and+spelling+rules#Terms,abbreviations,andspellingrules-B)